front/src/reducers/__tests__/osrdconf.spec.ts

```diff
@@ -7,166 +7,231 @@ import {
   toggleOriginLinkedBounds,
 } from '../osrdconf';
 
-const createStore = (initialStateExtra?: Partial<OsrdConfState>) =>
-  createStoreWithoutMiddleware({ osrdconf: { ...initialState, ...initialStateExtra } });
+const simulationConfInitialState: Partial<OsrdConfState> = initialState.simulationConf;
+const stdcmConfInitialState = initialState.stdcmConf;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const createStore = (initialStateExtra?: any) =>
```

Comment:
```suggestion
const createStore = (initialStateExtra?: Partial<RootState>) =>
```

Reply:
no it does not work, lead to a ts nightmare. Try it out maybe you'll find a solution

Reply:
Ok, we should try to find some time to fix this :+1: